tic-tac-toe/Rules.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+
+namespace tic_tac_toe
+{
+    public static class Rules
+    {
+        public static bool GameOver(Board board) => HasWinner(board) || board.IsFull();
```

Comment:
I like this syntax 👍  https://csharp.2000things.com/2014/11/03/1217-using-lambda-expressions-for-function-members/

Reply:
Yeah, and the best thing is that `command + .` in VS Code offers this refactoring from function body to expression body. I've coded in Vim for years now but I'm starting to really get into using those automated editor shortcuts in VS Code 😂  🤯